test/functional/mempool_packages.py

```diff
@@ -64,9 +75,9 @@ def run_test(self):
         descendant_fees = 0
         descendant_vsize = 0
 
-        ancestor_vsize = sum([mempool[tx]['vsize'] for tx in mempool])
+        assert_equal(ancestor_vsize, sum([mempool[tx]['vsize'] for tx in mempool]))
         ancestor_count = MAX_ANCESTORS
-        ancestor_fees = sum([mempool[tx]['fee'] for tx in mempool])
+        assert_equal(ancestor_fees, sum([mempool[tx]['fee'] for tx in mempool]))
```

Comment:
I don't think you can replace an assignment with `assert_equal`

Reply:
This will result in a run time error

Reply:
It's calculated above: https://github.com/bitcoin/bitcoin/pull/12677/files#diff-a01d25e7f69f4199a300a53bb4ff9957R56

(but yes, I need to update the size one to vsize...)

Reply:
Fixed